scripts/get-members.ts

```diff
@@ -0,0 +1,33 @@
+import { db } from './firebase';
+
+let getNetId = (email: string): string => {
+  let pos = email.search('@');
+  return email.slice(0, pos);
+};
+
+const fs = require('fs');
+const dirPath: string = 'scripts/members/';
+fs.mkdirSync(dirPath, { recursive: true });
+
+const jsonFilesList: string[] = fs.readdirSync(dirPath);
+jsonFilesList.forEach((json) =>
+  fs.unlink(dirPath + json, (err) => {
+    if (err) {
+      console.log(err);
+    }
+  })
+);
```

Comment:
what's fs.readdirSync() and what's fs.unlink? Is this a list of the json file names?

Reply:
`fs.readdirSync()` reads all the files in `dirPath` and `fs.unlink()` deletes a given file